app/build.gradle

```diff
@@ -98,10 +98,18 @@ dependencies {
     compile 'com.jakewharton.rxrelay:rxrelay:1.1.0'
 
     // Networking
-    compile 'com.squareup.retrofit:retrofit:1.9.0'
-    compile 'com.squareup.okhttp:okhttp:2.7.5'
+    compile 'com.squareup.okhttp3:okhttp:3.4.1'
```

Comment:
Changing to OkHttp3 means that the `InstrumentationModule` should use the new `StethoInterceptor` too.